edx_sga/sga.py

```diff
@@ -70,7 +70,7 @@ class StaffGradedAssignmentXBlock(XBlock):
         scope=Scope.settings
     )
 
-    points = Float(
+    points = Integer(
         display_name="Maximum score",
         help=("Maximum grade score given to assignment by staff."),
         values={"min": 0, "step": .1},
```

Comment:
Actually, it looks like I figure out how to do this in the save handler with some error checking on the save handler.